Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1164,7 +1168,7 @@ void TextureCache::TCacheEntryBase::SetEfbCopy(u32 stride)
 	is_efb_copy = true;
 	memory_stride = stride;
 
-	_assert_msg_(VIDEO, memory_stride >= CacheLinesPerRow(), "Memory stride is too small");
+	_assert_msg_(VIDEO, memory_stride >= BytesPerRow(), "Memory stride is too small");
```

Comment:
Note the behavior change here. I had asked degasus on IRC about this

> [11:53:09]  \<LioncashMBP\>	degasus: Might have to make it a separate function, because it's used directly here: https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/VideoCommon/TextureCacheBase.cpp#L1167
> [11:54:10]  \<degasus\>	LioncashMBP: likely used wrong :/
> [11:54:36]  \<degasus\>	at least size_in_bytes != size_in_cache_lines